internal/gopsutilenv/testdata/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
+*
+# Except this .gitignore file
+!.gitignore
```

Comment:
Why is this file needed?

Reply:
need a path for testing , this is the best practice to create empty dir in git